src/frontend/running.rs

```diff
@@ -52,21 +66,27 @@ struct FarmerState {
     token_symbol: String,
 }
 
+#[allow(dead_code)]
 #[derive(Debug)]
 pub struct RunningView {
     node_view: Controller<NodeView>,
     node_synced: bool,
     farmer_state: FarmerState,
     farms: FactoryHashMap<u8, FarmWidget>,
     plotting_paused: bool,
+    slot_probability: (u64, u64),
+    space_pledged: u128,
+    last_reward_timestamp: Option<u64>,
+    reward_eta_progress_bar: Controller<CircularProgressBar>,
+    reward_eta_progress_bar_moving: bool,
```

Comment:
Likely useful for debugging, but I don't think we need it in the repo

Reply:
- `space_pledged` is moved to `FarmerState`.
- `last_reward_timestamp` is removed.
- `reward_eta_progress_bar` is there to be used as a component.
- `reward_eta_progress_bar_moving` as a state to ensure if progress bar is moving then it will not recalculate the ETA, else it will.